packages/mobile/src/walletConnect/screens/Loading.tsx

```diff
@@ -13,7 +13,7 @@ import { isScreenOnForeground, navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { StackParamList } from 'src/navigator/types'
 
-const CONNECTION_TIMEOUT = 10_000
+const CONNECTION_TIMEOUT = 20_000
```

Comment:
was this increased for the CI? i think 20 seconds is a long time for a user to wait (but then again, 10 seconds is already a long time) - my only thought is in mainnet we'd lose some of the wallet connect timeout analytics after 10 seconds if the user abandons the app after that time. not sure if this is a big issue, it could be fixed by moving this to config.ts and changing it for the test env if we want :)